admin/.dockerignore

```
@@ -0,0 +1,4 @@
+.git
+*Dockerfile*
+*docker-compose*
+node_modules
```

:thinking: Je ne comprends pas l'utilité de cette ligne
```suggestion
*Dockerfile*
*docker-compose*
```